lang/cs/Org.Apache.REEF.Network/Group/Operators/Impl/BroadcastReceiver.cs

```diff
@@ -79,6 +76,17 @@ public sealed class BroadcastReceiver<T> : IBroadcastReceiver<T>
         public string OperatorName { get; private set; }
 
         /// <summary>
+        /// Ensure all parent and children nodes in the topology are registered with teh Name Service.
+        /// </summary>
+        public void WaitForRegistration()
```

Comment:
It is defined in the interface, therefore must be public. Every sub class needs it. So must be defined in the interface. 

Another option is to add an abstract layer to put common stuff of the sub classes and remove it from the interface. There are a lot can be moved to a base layer. If that sounds a good way, I can do refactor in a separate Jira.